DataFormats/DeepFormats/BuildFile.xml

```diff
@@ -0,0 +1,14 @@
+<use   name="DataFormats/Common"/>
```

Comment:
I think that the content of the FeepFormats package should be just moved to DataFormats/BTauReco

Reply:
@slava77 that is relatively easy to do, and I see also the drawback of adding a new subpackage but it was a bit cleaner to keep it separate for development. Does everyone else agree on this?

Reply:
done with https://github.com/cms-sw/cmssw/pull/19893/commits/92e58c5b9bcb81891f0492c8fe39fbf741f5b9d3 and https://github.com/cms-sw/cmssw/pull/19893/commits/435c7b4d95e101457c93b043d46c1852c3fa064f